src/vmaware.hpp

```diff
@@ -10352,41 +9761,39 @@ struct VM {
             flag_collector.reset();
             generate_default(disabled_flag_collector);
 
-            if VMAWARE_CONSTEXPR (is_empty<Args...>()) {
+            if constexpr (is_empty<Args...>()) {
```

Comment:
i added `VMAWARE_CONSTEXPR` because `if constexpr` is only available on C++17, and the macro was used to handle this by having it empty if CPP < 17, and have it "constexpr" if CPP >= 17